Sprint-3/revise/implement/get-ordinal-number.test.js

```diff
@@ -2,3 +2,43 @@
 
 // continue testing and implementing getOrdinalNumber for additional cases
 // Write your tests using Jest - remember to run your tests often for continual feedback
+
+// Import the function to test
```

Comment:
you know what to do with these comments ;)

Reply:
Thanks for the feedback! I've added more test cases and continued implementing the function to cover additional scenarios as suggested.

Reply:
Thanks. Please remove the repeated commented lines below. It is a good practice as unneeded comments clutter the file and make it harder to follow

Reply:
I did delete them however they were instructions of doing the exercises.

Reply:
I meant the repeated lines:
`/ Import the function to test

// get-ordinal-number.test.js`
These were dead code, and not part of the original instructions

Reply:
Oh I see. You are right. Sure I will do.